luigi_pipeline/lib/model/sv_mt_schema.py

```diff
@@ -19,9 +19,24 @@
 CONSEQ_PREDICTED_PREFIX = 'PREDICTED_'
 NON_GENE_PREDICTIONS = {'PREDICTED_INTERGENIC', 'PREDICTED_NONCODING_BREAKPOINT', 'PREDICTED_NONCODING_SPAN'}
 
+PREVIOUS_GENOTYPE_N_ALT_ALLELES = hl.dict({
+    # Map of concordance string -> previous n_alt_alleles()
 
-INTERVAL_TYPE = 'array<struct{type: str, chrom: str, start: int32, end: int32}>'
+    # Concordant
+    frozenset(["TN"]): 0,       # 0/0 -> 0/0
```

Comment:
I opted for a `frozenset` here because it's not clear to me whether the array of classifications is guaranteed to be ordered.  In practice, I think it _is_ ordered... the VCF always has `TP,TN` and never `TN,TP`.  In the slideshow though... it's the opposite:

<img width="92" alt="Screenshot 2023-03-29 at 3 12 24 PM" src="https://user-images.githubusercontent.com/5325727/228643613-2a00b6fe-42f0-486f-98f5-a64ddd24cefc.png">